test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -999,6 +1002,57 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 		framework.ExpectNoError(WaitForClusterSizeFunc(ctx, f.ClientSet,
 			func(size int) bool { return size == increasedSize }, time.Second))
 	})
+
+	f.It("should scale up when unprocessed pod is created and is going to be unschedulable", feature.ClusterScaleUpBypassScheduler, func(ctx context.Context) {
+		schedulerName, found := framework.TestContext.ExtraParams[nonExistingBypassedSchedulerNameKey]
+		if !found {
+			framework.Logf("Skipping test, Didn't find an ignored non-existent scheduler name to use")
+			return
+		}
```

Comment:
don't implement conditional tests, the jobs should be skipping the test based on the tags